drools-wb-screens/drools-wb-enum-editor/drools-wb-enum-editor-client/src/test/java/org/drools/workbench/screens/enums/client/editor/EnumEditorPresenterTests.java

```diff
@@ -94,6 +101,12 @@
     @Mock
     private Overview overview;
 
+    @Mock
+    private WorkspaceProjectContext mockWorkbenchContext;
```

Comment:
I'm curious why this is needed? I don't see how your changes would need it.

Reply:
In fact, this was broken. The tests were "passing" because the name of the class was `EnumEditorPresenterTests` instead of `EnumEditorPresenterTest` heheh